go.mod

```diff
@@ -38,17 +38,17 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/rancher/dynamiclistener v0.5.0-rc2
 	github.com/rancher/lasso v0.0.0-20240123150939-7055397d6dfa
-	github.com/rancher/rancher/pkg/apis v0.0.0-20240328110445-91a4620d7e49
-	github.com/rancher/rke v1.5.7-rc2
+	github.com/rancher/rancher/pkg/apis v0.0.0-20240506132424-2b53406a5497
+	github.com/rancher/rke v1.6.0-rc1
 	github.com/rancher/wrangler/v2 v2.1.4
 	github.com/sirupsen/logrus v1.9.3
-	github.com/stretchr/testify v1.8.4
+	github.com/stretchr/testify v1.9.0
 	golang.org/x/exp v0.0.0-20230817173708-d852ddb80c63
 	golang.org/x/text v0.14.0
-	golang.org/x/tools v0.16.1
-	k8s.io/api v0.28.6
-	k8s.io/apimachinery v0.28.6
-	k8s.io/apiserver v0.28.6
+	golang.org/x/tools v0.17.0
+	k8s.io/api v0.28.8
+	k8s.io/apimachinery v0.28.8
+	k8s.io/apiserver v0.28.8
```

Comment:
those were changed because I bumped the fleet version in order to have the ExternalRules field. This is not needed anymore since the k8s 1.29 PR was merged, which was also bumping the rancher version. No go.mod changes after fixing the conflicts